api/src/main/java/org/eclipse/microprofile/config/spi/ConfigSource.java

```diff
@@ -179,7 +178,18 @@ default int getOrdinal() {
      * @param propertyName the property name
      * @return the property value, or {@code null} if the property is not present
      */
-    String getValue(String propertyName);
+    default String getValue(String propertyName) {
```

Comment:
we did not do this due to performance implication.
This also kind of turns the whole design upside down.
The ConfigValue is supposed to be ON TOP of the 'native raw config' which `getProperties` and `getValue` represent.

Reply:
It's fine, we can remove the default implementation.